src/main/scala/interplay/PlayBuildBase.scala

```diff
@@ -296,11 +308,17 @@ object PlayRootProjectBase extends AutoPlugin {
   import PlayBuildBase.autoImport._
 
   override def projectSettings = PlayNoPublishBase.projectSettings ++ Seq(
+    // In sbt 1 the scripted plugin is automatically enabled. It imports
+    // its own jars into the project as a dependency. This fails for Scala
+    // 2.11 projects so we use this hack to remove the JARs again. :/
+    libraryDependencies := libraryDependencies.value.filter { m =>
+      m.organization != "org.scala-sbt" && !(m.name == "scripted-sbt" || m.name == "sbt-launch")
+    },
     crossScalaVersions := {
       if ((playCrossBuildRootProject in ThisBuild).?.value.exists(identity)) {
-        Seq(ScalaVersions.scala211, ScalaVersions.scala212)
+        Seq(playBuildScala211Version.value, playBuildScala212Version.value)
       } else {
-        Seq(ScalaVersions.scala210, ScalaVersions.scala212)
+        Seq(playBuildScala210Version.value, playBuildScala212Version.value)
```

Comment:
I had a typo on this line before - I had `playBuildScala211Version` instead of `playBuildScala210Version`. Tests now pass with this change.